### DISCUSSION TITLE
Structure about "platform.vsphere.folder" in vSphere provising unclear

### DISCUSSION THREAD

**Discussion by danielchristianschroeter** [okd-project/okd#1225] [Q&A] [answered]
In https://docs.okd.io/latest/installing/installing_vsphere/installing-vsphere-installer-provisioned-customizations.html you are able to define a path within your vSphere cluster.
In the example of platform.vsphere.folder it shows the following strict structure "/<datacenter_name>/vm/<folder_name>/<subfolder_name>"
My real target path is "/DATACENTER/ABC/Test/OKD/abc" I can not find any /vm/ in the vCenter UI. This /vm seems to be required:
FATAL failed to fetch Metadata: failed to load asset "Install Config": invalid "install-config.yaml" file: platform.vsphere.folder: Invalid value: "/DATACENTER/ABC/Test/OKD/abc": folder must be absolute path: expected prefix /DATACENTER/vm/ 

If I add the /vm to the Path "/DATACENTER/vm/ABC/Test/OKD/abc" the location can not be found:
FATAL failed to fetch Terraform Variables: failed to fetch dependency of "Terraform Variables": failed to generate asset "Platform Provisioning Check": [platform.vsphere.folder: Invalid value: "/DATACENTER/vm/ABC/Test/OKD/abc": folder '/DATACENTER/vm/ABC/Test/OKD/abc' not found.

Where I can get the real path the openshift-installer requires? It seems the path differs from the vCenter UI.

**Comment by jcpowermac** [comment, marked as answer]
@myg0v the `vm` folder is hidden but is there - you can use the mob (https://your-vcenter-url/mob) to see it

Since we use govmomi as the sdk I recommend using `govc` (https://github.com/vmware/govmomi) for determining the path.

**Reply by danielchristianschroeter** [reply]
Thank you!

govc vm.info shows the required path.
